### PR TITLE
fix #39534 sikisonline.club

### LINKED ISSUE #39534: sikisonline.club
### Issue URL (Annoyance)
[NSFW][http://sikisonline.club/72-full-hd-yerli-turkce-porn-izle.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fsikisonline.club%2F72-full-hd-yerli-turkce-porn-izle.html)
### Comment
Username: @VivaLaVidaValeVoua
> 
> --
> Aşağıdaki bilgiler otomatik olarak oluşturuldu. Silmeyin.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js
> https://adsbypasser.github.io/releases/adsbypasser.full.es5.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.2.2903.0
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Self-destructing third-party cookies (0), Block WebRTC, Block Push API, Block Location API
Filters: | EasyList,<br>AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard Base,<br>Adblock Warning Removal List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/39534

### CODE REVIEW

**Review by BlazDT — COMMENTED** [review]
Site does not open currently for me, so I can't check, though rules look ok.

File: TurkishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@
 ||adreman.pro^$third-party
 ||turkakisi.com^$third-party
 ||adrequests.com^$third-party
+||wapempire.com^
```
Comment (Alex-302) [inline TurkishFilter/sections/adservers.txt]: tracker. $third-party

File: AnnoyancesFilter/sections/push-notifications.txt [hunk 2]
```diff
@@ -22,6 +22,7 @@ macwelt.de,pcwelt.de#%#AG_setConstant('firebase.messaging', 'noopFunc');
 !
 ||list-news.net^$third-party
 ||tutad.ru^
+||wwsercher.biz^
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/push-notifications.txt]: $third-party

**Comment by zloyden** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/39906

**Pull request closed without merging**